companyweb_base/views/partner_view.xml

```diff
@@ -0,0 +1,267 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="companyweb_partner_view" model="ir.ui.view">
+        <field name="name">partner.cweb_info</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="country_id" position="after">
+                <p>
+                    <div>
+                        <field invisible="1" name="cweb_show_button_enhance" />
+                        <button
+                            aria-label="Enhance"
+                            class="btn-sm btn-link "
+                            icon="companyweb_base/static/img/logo-en.resized.png"
```

Comment:
It was understood in earlier version but not anymore <br/>
could you change to "/companyweb_base/static/img/logo-en.resized.png" (the first / is missing)